graph/ShortestWay/ShortestWay.h

```diff
@@ -0,0 +1,36 @@
+//
+// Created by emyrmak on 7/9/2019.
+//
+#pragma once
+#include <limits>
+#include <vector>
+#include "Vertices.h"
+
+using Matrix = std::vector<std::vector<int>>;
+using intVector = std::vector<int>;
+using boolVector = std::vector<bool>;
+
+class ShortestWay  {
+public:
+    ShortestWay (Vertices& vertices);
+    ~ShortestWay ();
+    void findShortestWay(int startPoint, int endPoint);
+private:
+    const int maxInt = std::numeric_limits<int>::max();
+    const std::size_t size;
+    const int startNodeIndex;
```

Comment:
constant variables that are not related to the particular class instance are better of defined as constexpr or static consts before the class definition.